main.cpp

```diff
@@ -380,18 +533,114 @@ Part 1e - Step 19: Request a review
 paste your code below
 */
 
+int main()
+{
 
 
+    Bicycle purple; //Note to self: I am instantiating a UDT named 'purple' in the main()
+    purple.transportPerson(); //Note to self:  I am calling the member functions of the UDT instance
+    purple.rollDownhill();
+    purple.repairs();
 
+    std::cout << "Is purple's member variable 'rides' equal to 5? " << (purple.rides == 5 ? " Yes" : " No") << "\n" << std::endl;
 
 
+    RecordingStudio soundbylaura;//Note to self: Creates an instance of the RecordingStudio struct named soundbylaura.
+    RecordingStudio::ControlRoom controlRoomB;//Note to self: Creates an instance of the nested RecordingStudio::ControlRoom struct named controlRoomB.
 
+    controlRoomB.bookRoom( true, 5);
+    controlRoomB.prepareRoom( true, true, 1, 5.0f); 
+    controlRoomB.caclulateTotalFee( 12.0f, 650.50f, 4.0f);
+    //Note to self: Don't forget, calling functions that have arguments here expect direct values, not type declarations.
 
+    std::cout << "Is the room being prepped?" << (controlRoomB.isBooked == true ? " Yes" : " No") << "\n" << std::endl;   
 
-int main()
-{
-    Example::main(); 
+    soundbylaura.beginRecordingSession( controlRoomB);
+    soundbylaura.startRateClock( true);
+    soundbylaura.closeRoom( controlRoomB, true);
+    soundbylaura.billClient( controlRoomB, 12, 3, 3);
+
+    soundbylaura.hostEvent();
+    soundbylaura.recordAudio();
+    soundbylaura.sendInvoice();
+
+    std::cout << "Are we hosting an event today:" << (soundbylaura.employees == 0 ? " Yes" : " No") << "\n" << std::endl;
+
+    
+    AudioInterface studio1810C;
+    studio1810C.outputAudio();
+    studio1810C.receiveAudio();
+    studio1810C.displayLevels();
+
+    std::cout << "The type of audio output is: " << (studio1810C.outputType) << "\n " << std::endl;
+
+    
+    StereoWidenerAudioPlugin SBLWide;
+    StereoWidenerAudioPlugin::MixKnob mix;
+    
+    SBLWide.captureAudio();
+    SBLWide.textInfo();
+    SBLWide.widenSignal();
+
+    SBLWide.increaseWetness( mix);
+    SBLWide.decreaseWetness( mix);
+    SBLWide.smartMute( 1, 1.1f);
+
+    std::cout << "This knob color should be: " << (SBLWide.knobsColors) << "\n" << std::endl;
+
+    mix.getStateInformation(int (256), float (0.0f));
+    mix.prepareToPlay( 44100.0, 256.0f);
+    mix.isBypassed( false, false);
+
+    std::cout << "The name of this knob should be: " << (mix.label) << "\n" << std::endl;
+
+    
+    GraphicalUserInterface darkMode;
+    darkMode.displayInputLevel();
+    darkMode.displayAttenuation();
+    darkMode.parameterMod();
```

Comment:
Function names should include a verb, because they represent actions.  
- [ ] revise